packages/ui/src/components/Dropdown/dropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import { Dropdown as DefaultDropdown, DropdownProps as DefaultDropdownProps } from 'antd'
+import { isFunction } from 'lodash'
+import get from 'lodash/get'
+import React, { useCallback } from 'react'
+
+import { extractChildrenContent, sendAnalyticsClickEvent } from '../_utils/analytics'
+
+
+export const DROPDOWN_CLASS_PREFIX = 'condo-dropdown'
+
+export type DropdownProps = Omit<DefaultDropdownProps, 'visible' | 'prefixCls' | 'overlayStyle'>
```

Comment:
probably `style` too?

Reply:
No, this is the only property with a style